<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resolución de Problema</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_Main">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    
    <item row="0" column="0">
     <widget class="QWidget" name="gameContentContainer" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frameTools">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutTools">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>5</number>
          </property>
          <item>
           <widget class="QLabel" name="labelDrawTitle">
            <property name="text">
             <string>Dibujo</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPunto">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Marcar un punto</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/punto.svg</normaloff>:/resources/icons/punto.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDrawLine">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Trazar línea</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/Linea_icon.svg</normaloff>:/resources/icons/Linea_icon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDrawArc">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Compás</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/compas_icon.svg</normaloff>:/resources/icons/compas_icon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnAnnotateText">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Texto</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/T.svg</normaloff>:/resources/icons/T.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_1">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelMeasureTitle">
            <property name="text">
             <string>Medición</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnProtractor">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Transportador</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/transportador.svg</normaloff>:/resources/icons/transportador.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRulerDistance">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Regla</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/ruler_icon.svg</normaloff>:/resources/icons/ruler_icon.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnChangeColor">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Color</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/colores.svg</normaloff>:/resources/icons/colores.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEraser">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Borrador</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/eraser.svg</normaloff>:/resources/icons/eraser.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClearMap">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Limpiar todo</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/ocultar.svg</normaloff>:/resources/icons/ocultar.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnShowCoordinates">
            <property name="minimumSize">
             <size>
              <width>60</width>
              <height>60</height>
             </size>
            </property>
            <property name="toolTip">
             <string>Coordenadas</string>
            </property>
            <property name="icon">
             <iconset resource="resources.qrc">
              <normaloff>:/resources/icons/coordenadas.svg</normaloff>:/resources/icons/coordenadas.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacerTools">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       
       <item>
        <layout class="QVBoxLayout" name="layoutMap">
         <item>
          <layout class="QHBoxLayout" name="layoutZoom">
           <item>
            <widget class="QLabel" name="lblZoomIcon">
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap>:/lupa.jpg</pixmap>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnZoomIn">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnZoomOut">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView">
           <property name="dragMode">
            <enum>QGraphicsView::ScrollHandDrag</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       
       <item>
        <widget class="QFrame" name="frameProblem">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayoutProblem">
          <item>
           <widget class="QLabel" name="labelTitle">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Problema:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelProblemText">
            <property name="text">
             <string>Texto del problema aquí...</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="Line" name="lineProblem">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAns1">
            <property name="text">
             <string>Opción A</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAns2">
            <property name="text">
             <string>Opción B</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAns3">
            <property name="text">
             <string>Opción C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioAns4">
            <property name="text">
             <string>Opción D</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacerProblem">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnCheck">
            <property name="text">
             <string>Comprobar Respuesta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="text">
             <string>Volver a Selección</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>

    <item row="0" column="0">
     <widget class="QWidget" name="selectionOverlay" native="true">
      <property name="styleSheet">
       <string>background-color: rgba(255, 255, 255, 225);</string>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayoutOverlay">
       
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutTopBar">
         <item>
          <spacer name="horizontalSpacerTop">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayoutUser">
           <item>
            <widget class="QPushButton" name="btnAvatar">
             <property name="minimumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>80</width>
               <height>80</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="iconSize">
              <size>
               <width>60</width>
               <height>60</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnLogout">
             <property name="text">
              <string>Cerrar sesión</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>

       <item>
        <spacer name="verticalSpacerTop2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>

       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnRandom">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>50</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>¡Problema aleatorio!</string>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QPushButton" name="btnToggleList">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Ver lista de problemas ▼</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QListWidget" name="listProblems">
         <property name="minimumSize">
          <size>
           <width>500</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>600</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>

       <item>
        <spacer name="verticalSpacerBottom">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>